File: C/Colleen.c
#include <stdio.h>

/*
This is a comment outside the main function
*/
void helper_function(const char *source)
{
    printf(source, 10, 34, source);
}

int main(void)
{
    /*
    This is a comment inside the main function
    */
    const char *source = "#include <stdio.h>%1$c%1$c/*%1$cThis is a comment outside the main function%1$c*/%1$cvoid helper_function(const char *source)%1$c{%1$c    printf(source, 10, 34, source);%1$c}%1$c%1$cint main(void)%1$c{%1$c    /*%1$c    This is a comment inside the main function%1$c    */%1$c    const char *source = %2$c%3$s%2$c;%1$c    helper_function(source);%1$c    return 0;%1$c}%1$c";
    helper_function(source);
    return 0;
}

File: bonus/Colleen.py
# This is a comment outside of the main function

def quine() -> None:
    # This is a comment inside the quine function
    s = "# This is a comment outside of the main function{0}{0}def quine() -> None:{0}    # This is a comment inside the quine function{0}    s = {1}{2}{1}{0}    print(s.format(chr(10), chr(34), s)){0}{0}{0}def main() -> None:{0}    quine(){0}{0}{0}if __name__ == {1}__main__{1}:{0}    main()"
    print(s.format(chr(10), chr(34), s))


def main() -> None:
    quine()


if __name__ == "__main__":
    main()
